dali/operators/random/coin_flip_gpu.cu

```diff
@@ -16,7 +16,7 @@
 #include <utility>
 #include "dali/core/convert.h"
 #include "dali/core/dev_buffer.h"
-#include "dali/kernels/alloc.h"
+#include "dali/core/mm/memory.h"
```

Comment:
Since there're no lines of code changed, does it mean that we don't really need this header here?

Reply:
Good spot. It was automatic find-replace.